--- app/src/main/java/com/henryliu/cbtreframe/MainActivity.kt
+++ app/src/main/java/com/henryliu/cbtreframe/MainActivity.kt
@@ -2,7 +2,7 @@
 
 import android.os.Bundle
-import androidx.activity.ComponentActivity
+import androidx.fragment.app.FragmentActivity
 import androidx.activity.compose.setContent
@@ -15,7 +15,7 @@
 import com.henryliu.cbtreframe.shared.db.HistoryEntity
 
-class MainActivity : ComponentActivity() {
+class MainActivity : FragmentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
